README.md

```diff
@@ -1,37 +1,143 @@
-# Fluence browser client
+# Fluence JS SDK
 
 [![npm](https://img.shields.io/npm/v/@fluencelabs/fluence)](https://www.npmjs.com/package/@fluencelabs/fluence)
 
-Browser client for the Fluence network based on the js-libp2p.
+Official SDK for building web-based applications for [Fluence network](https://fluence.network/) 
 
-## How to build
+## Installation
 
-With `npm` installed building could be done as follows:
+With npm
 
 ```bash
-npm install fluence
+npm install @fluencelabs/fluence
 ```
 
-## Example
+With yarn
 
-Shows how to register and call new service in Fluence network.
+```bash
+yarn add @fluencelabs/fluence
+```
 
-Generate new peer ids for clients.
+## Getting started
+
+Initialize client
 
 ```typescript
-let peerId1 = await Fluence.generatePeerId();
-let peerId2 = await Fluence.generatePeerId();
+const relayNode = '/dns4/stage.fluence.dev/tcp/19001/wss/p2p/12D3KooWEXNUbCXooUwHrHBbrmjsrpHXoEphPwbjQXEGyzbqKnE9';
+
+let clinet: FluenceClient;
+
+createClient(relayNode);
+	.then((c) => {
+		client = c;
+    })
+	.catch((err) => console.log('Client initialization failed', err));
 ```
 
-Establish connections to predefined nodes.
+Make a particle
 
 ```typescript
-let client1 = await Fluence.connect(
-    '/dns4/134.209.186.43/tcp/9003/ws/p2p/12D3KooWBUJifCTgaxAUrcM9JysqCcS4CS8tiYH5hExbdWCAoNwb',
-    peerId1,
-);
-let client2 = await Fluence.connect(
-    '/ip4/134.209.186.43/tcp/9002/ws/p2p/12D3KooWHk9BjDQBUqnavciRPhAYFvqKBe4ZiPPvde7vDaqgn5er',
-    peerId2,
+const particle = new Particle(`
+    (seq
+        (call myRelay ("op" "identity") [])
+        (call userlistNode (userlist "join") [user])
+    )`,
+    {
+        myRelay: client.relayPeerId,
+        myPeerId: client.selfPeerId,
+        user: {
+            name: 'john',
+            peer_id: client.selfPeerId,
+            relay_id: client.relayPeerId,
+        },
+        userlist: '03edcc81-7777-4234-b048-36305d8d65e2',
+        userlistNode: '12D3KooWEXNUbCXooUwHrHBbrmjsrpHXoEphPwbjQXEGyzbqKnE9',
+    },
 );
 ```
+
+Send it to the network
+
+```typescript
+await sendParticle(client, particle);
+```
+
+Respond service function calls
+
+```typescript
+subscribeToEvent(client, 'helloService', 'helloWorld', (args) => {
+    const [message] = args as [string];
+    console.log(message);
+});
+```
+
+## Documentation
+
+Detailed guide:  [readme.io](https://fluence-labs.readme.io/docs/build-an-app)
```

Comment:
maybe use: https://fluence-labs.readme.io/docs/js-sdk
so we don't land on a TBD page :)